pkg/services/provisioning/provisioning.go

```diff
@@ -182,7 +184,6 @@ func (ps *ProvisioningServiceImpl) Run(ctx context.Context) error {
 	err := ps.ProvisionDashboards(ctx)
 	if err != nil {
 		ps.log.Error("Failed to provision dashboard", "error", err)
-		return err
```

Comment:
Ideally, I'd suggest to keep a whilelist of "safe" errors, such as a non-existent parent folder. For any other unknown error I'd rather stop Grafana. Otherwise it's a notable breaking change in behaviour.

Reply:
Thanks for the input! The issue though is that any provisioning error would then prevent grafana from starting successfully. Provisioning is brittle and there could be errors besides a non-existent parent folder which also wouldn't justify grafana not starting.

Question about logistics when backporting. Let's say I implemented your suggestion, how would I properly backport the change? For context, the original PR has already been merged: https://github.com/grafana/grafana/pull/92201. I can still open another PR to implement your suggestion. Would I then open a single backport PR that references both changes?